### PR TITLE
fix: recommended block content issues EXLM-2167

### PR DESCRIPTION
**Pull request by nitin-rachabathuni**
Please always provide the Jira Issue your PR is for, as well as test URLs where your change can be observed (before and after):

Jira ID: https://jira.corp.adobe.com/browse/EXLM-2167

Test URLs:

- Before: https://main--exlm--adobe-experience-league.hlx.live?martech=off
- After: https://fix-exlm-2167--exlm--adobe-experience-league.hlx.live?martech=off

### CODE REVIEW

File: blocks/recommended-content/recommended-content.css [hunk 1]
```diff
@@ -33,6 +33,47 @@
     align-items: center;
 }
 
+.recommended-content-see-new {
```
Comment (noormd08) [inline blocks/recommended-content/recommended-content.css:36]: rename this class with proper one.

File: blocks/recommended-content/recommended-content.js [hunk 2]
```diff
@@ -62,14 +62,18 @@ export default async function decorate(block) {
   block.appendChild(blockHeader);
 
   const isDesktop = window.matchMedia('(min-width:900px)').matches;
+  const reversedDomElements = remainingElements.reverse();
+  const [seeWhatsNew, discoverResources, sortingEl, ...rolesAndOtherElements] = reversedDomElements;
```
Comment (noormd08) [inline blocks/recommended-content/recommended-content.js:66]: Dont use the variable names with specific text, as it may vary in later stage.

Comment (nitin-rachabathuni) [reply blocks/recommended-content/recommended-content.js]: done Noor

File: blocks/recommended-content/recommended-content.css (hunk 1 above)
Comment (nitin-rachabathuni) [reply blocks/recommended-content/recommended-content.css]: done Noor

**Review by noormd08 — APPROVED** [review]

**Pull request merged**